scripts/README.md

```diff
@@ -0,0 +1,34 @@
+# DuckDB-Web scripts
+
+## `generate_all_docs.sh`
+
+### Prerequisites
+
+Build DuckDB in `debug` mode. Install the Python client as well as the `httpfs` and `icu` extensions:
+
+```bash
+# build the DuckDB CLI client
+GEN=ninja BUILD_HTTPFS=1 BUILD_ICU=1 OPENSSL_ROOT_DIR=/opt/homebrew/bin/ make debug
```

Comment:
What's the rationale behind directing people to build from scratch instead of installing dev builds from pypi?

Reply:
To be frank, it didn't occur to me to use dev builds in the first place. That's indeed much simpler and definitely the recommended approach.

Reply:
Pushed a commit that recommends the nightly build.